ui/core/src/ui-core/inputs/numericinput/NumberInput.scss

```diff
@@ -0,0 +1,71 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+@import "../../base/base";
+
+.core-numeric-input-container {

```

Comment:
".core-number-input" should probably be used here instead.

Reply:
Renamed all occurrences.